site/content/en/topics/node-types.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Support Multiple Node Types"
+linkTitle: "Node Types"
+weight: 4
+type: docs
+description: >
+    Node type description in provider azure.
+---
+
+Kubernetes v1.26 adds support for using [Azure VMSS Flex VMs](https://learn.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-orchestration-modes#scale-sets-with-flexible-orchestration) as the cluster nodes. Besides, mixing up different VM types in the same cluster is also supported. There is no API change expected from E2E users' perspective when manipulating the Kubernetes cluster, however, users can choose to specify the VM type when configuring the Cloud Provider to further optimize the API calls in Cloud Controller Manager. Below are the configurations suggested based on the cluster modes.
```

Comment:
E2E users -> end users?
Could you explain more about how to `further optimize the API calls` by configuring the Cloud Provider?

Reply:
Sure, I added it to the comments column.